TicTacToe2/TicTacToeGame.cs

```diff
@@ -9,11 +9,12 @@ class TicTacToeGame
         public void Play()
         {
             bool winner = false;
-            while (!winner)
+            bool draw = false;
+            while (!winner || !draw)
```

Comment:
A different logical operator should be used here. || represents "or"

T || T = T
F || T = T
T || F = T
F || F = F